src/libcharon/plugins/stroke/stroke_config.c

```diff
@@ -418,14 +418,30 @@ static auth_cfg_t *build_auth_cfg(private_stroke_config_t *this,
 		enumerator_t *enumerator;
 		bool has_subject = FALSE;
 		certificate_t *first = NULL;
+		certificate_t *last = NULL;
 
 		enumerator = enumerator_create_token(cert, ",", " ");
 		while (enumerator->enumerate(enumerator, &cert))
 		{
 			certificate = this->cred->load_peer(this->cred, cert);
 			if (certificate)
 			{
-				cfg->add(cfg, AUTH_RULE_SUBJECT_CERT, certificate);
```

Comment:
What exactly is this used for?

Reply:
eap-anyconnect at certain point expect to send whole trustchain. Unfortunately the IM certificates are not loaded anywhere, therefore I have added this option to stroke (and vici). I will put it in separate commit